### ISSUE TITLE
tlx.nn.BatchNorm中act参数下没有Swish激活函数

### ISSUE THREAD

**Issue by Eric-yuye** [tensorlayer/TensorLayerX#41]
tlx.nn.BatchNorm中act参数下没有Swish激活函数
